PGIP/Output/Proof.hs

```diff
@@ -108,12 +109,16 @@ formatProofs format options proofs = case format of
         _ -> error $ "Failed reading the number " ++ show (todSec tod)
     }
 
-  convertTacticScript :: LP.ProofStatus G_proof_tree -> TacticScript
-  convertTacticScript ps =
-    let atp = read $ (\ (LP.TacticScript ts) -> ts) $ LP.tacticScript ps
-    in TacticScript { timeLimit = tsTimeLimit atp
-                    , extraOptions = tsExtraOpts atp
-                    }
+  convertTacticScript :: Maybe (LP.ProofStatus G_proof_tree)
+                      -> Maybe TacticScript
+  convertTacticScript Nothing = Nothing
+  convertTacticScript (Just ps) =
+    let tsStr = (\ (LP.TacticScript ts) -> ts) $ LP.tacticScript ps
+    in case readMaybe tsStr of
```

Comment:
let-in should be removed

```
case (\ (LP.TacticScript ts) -> readMaybe ts) $ LP.tacticScript ps of
```

(Also record syntax for LP.TacticScript in LP could be used to get a proper selector.)